website/docs/docs/build/dimensions.md

```diff
@@ -173,28 +159,28 @@ measures:
 
 <TabItem value="time_gran" label="time_granularity">
 
-`time_granularity` specifies the smallest level of detail that a measure or metric should be reported at, such as daily, weekly, monthly, quarterly, or yearly. Different granularity options are available, and each metric must have a specified granularity. For example, a metric specified with weekly granularity couldn't be aggregated to a daily grain. 
+`time_granularity` specifies the grain of a time dimension. MetricFlow will transform the underlying column to the specified granularity. For example, if you add hourly granularity to a time dimension column, MetricFlow will run a `date_trunc` function to convert the timestamp to hourly. You can easily change the time grain at query time and aggregate it to a coarser grain, for example, from hourly to monthly. However, you can't go from a coarser grain to a finer grain (monthly to hourly).
 
-The current options for time granularity are day, week, month, quarter, and year. 
+Any granularity supported by your engine's `date_trunc` function will work, with the most common granularities being hour, day, week, month, quarter, and year.
```

Comment:
This isn't quite accurate (e.g., [look at the options available for snowflake](https://docs.snowflake.com/en/sql-reference/functions-date-time#label-supported-date-time-parts)). Might be better to just list the options we support.
For sub-daily options, we support these for all engines unless otherwise noted):
- nanosecond (snowflake only)
- microsecond (all engines except trino)
- millisecond
- second
- minute
- hour

Reply:
Updated